pkg/devfile/parser/parse.go

```diff
@@ -74,6 +75,22 @@ func ParseFromURL(url string) (d DevfileObj, err error) {
 	return parseDevfile(d)
 }
 
+// ParseFromData func parses and validates the devfile integrity.
+// Creates devfile context and runtime objects
+func ParseFromData(data []byte) (d DevfileObj, err error) {
```

Comment:
This function should have a parse and validate function like the others e.g ParseFromURLAndValidate()

Reply:
This is intentional and this is also it doesn't have `Validate` it the name.

Reply:
Thanks for adding ParseFromDataAndValidate() :+1: